spring clean user list

Mic92 opened the pull request.

mmisono commented:
- Moritz Beckel (https://github.com/TUM-DSE/doctor-cluster-config/pull/528/commits/05742c8a2e53c8362de16f8e5330b6636497e795) : He needs to have access until the end of April
- Robert Schambach (https://github.com/TUM-DSE/doctor-cluster-config/pull/528/commits/8e62d591064481a5099ae12321dcaf0f2b2b33f2): Can we keep his account until the end of May? as he might help some experiments. we can give him access rights only to vislor.

Mic92 commented:
> * Moritz Beckel ([05742c8](https://github.com/TUM-DSE/doctor-cluster-config/commit/05742c8a2e53c8362de16f8e5330b6636497e795)) : He needs to have access until the end of April
> 
>     * Robert Schambach ([8e62d59](https://github.com/TUM-DSE/doctor-cluster-config/commit/8e62d591064481a5099ae12321dcaf0f2b2b33f2)): Can we keep his account until the end of May? as he might help some experiments. we can give him access rights only to vislor.

I updated their deletion dates and removed them from this list.

cmainas commented on modules/users/students.nix:
```diff
@@ -290,66 +206,6 @@ in
-
-    # Martin Lambeck, Ma Thesis with Jiyang (FAIR)
-    # Remove after SS23
-    martinL = {
```
Let's keep Martin for one more semester, since he is helping with the FPGA serverless submission

Mic92 replied:
Added martinL back.

cmainas commented on modules/users/students.nix:
```diff
@@ -362,29 +218,6 @@ in
 
-    # Bruno Scheufler, Project work with Atsushi (Serverless FPGA)
-    # Remove after SS23
-    bruno = {
```
Let's keep Bruno for one more semester, since he is helping with the FPGA serverless submission

Mic92 replied:
Added Bruno back.

cmainas reviewed (commented):
Bruno and Martin are still woking with us in the FPGA serverless project. We can keep them for the time being.
